src/event.js

```diff
@@ -366,40 +366,41 @@ jQuery.event = {
 		//
 		// Support: Firefox <=42
 		// Avoid non-left-click in FF but don't block IE radio events (#3861, gh-2343)
-		if ( delegateCount && cur.nodeType &&
-			( event.type !== "click" || isNaN( event.button ) || event.button < 1 ) ) {
+		if ( delegateCount && cur.nodeType && !( event.type === "click" && event.button >= 1 ) ) {
```

Comment:
@markelog yeah, the comment mentions gh-2434 and In c82a6685bb964627e27008e298f93ea81218265b we changed this line because of arrow keys in IE11. How about adding something like this @gibson042 based on my [comment there](https://github.com/jquery/jquery/issues/2343#issuecomment-104758437)?
```js
// Support: IE11
// IE reports button==-1 for "clicks" on radios via the arrow keys
```